docker/base.dockerfile

```diff
@@ -1,5 +1,4 @@
 # Stages to extract toolchain and sysroot
-FROM faasm/cpp-sysroot:0.0.21 as sysroot
 FROM faasm/cpython:0.0.10 as python
```

Comment:
Unfortunately we still depend on the Python files at runtime, so have to copy them in using a multi-stage build